bundles/org.openhab.core.addon.marketplace/src/main/java/org/openhab/core/addon/marketplace/internal/community/CommunityMarketplaceAddonService.java

```diff
@@ -215,7 +215,7 @@ protected List<Addon> getRemoteAddons() {
 
         // retrieve from remote
         try {
-            URL url = new URL(String.format("%s%s", COMMUNITY_TOPIC_URL, uid.replace(ADDON_ID_PREFIX, "")));
+            URL url = URI.create(String.format("%s%s", COMMUNITY_TOPIC_URL, uid.replace(ADDON_ID_PREFIX, ""))).toURL();
```

Comment:
While you're there, `String.format("%s%s", COMMUNITY_TOPIC_URL, uid.replace(ADDON_ID_PREFIX, ""))` could be replaced by `"%s%s".formatted(COMMUNITY_TOPIC_URL, uid.replace(ADDON_ID_PREFIX, ""))`

Reply:
Done

Reply:
Or `COMMUNITY_TOPIC_URL + uid.replace(ADDON_ID_PREFIX, "")` 😉

Reply:
Obviously